### DISCUSSION TITLE
Why does Bisq need a new GUI?

### DISCUSSION THREAD

**Discussion by ghost** [bisq-network/bisq#5371] [General]
I think it doesn't.  I'm not vehemently against the UI being redesigned.  I wish those doing it the best of luck, and will help where possible.  **But I think it is an unnecessary waste of money**, or money spent in the wrong direction.

There are well known problems with Bisq: 

- financial barriers to entry (4x costly BTC transactions per trade)
- technical barriers to entry (heavyweight application, uses lots of memory, slow for many users, non-functional for some)
- reliability (number of failed trades)

Redesigning the UI does not fix any of those.

As a reminder, apparently revamping the UI will (per the project description) :

- Enable Bisq to capitalize on it's clear competitive advantages
- Improve user conversion rates
- Increase user loyalty and retention
- Increase growth in number of users
- Maximize revenue for the DAO
- Reduce support issues and associated costs
- Improve the trading experiences for users
- Enable is to be designed in a way that meets a variety of user needs and requirements

I do not see how it really solves any of these.
And finally I think the current UI is really very good.

**Comment by pazza83** [comment]
I agree that the most pressing issue is the financial barrier you reference. I also agree that the UI does not address any of three issues you reference. 

Although I made the [project proposal](https://github.com/bisq-network/projects/issues/49) I am also a fan of the current UI. I like how it is a little rough round the edges and seems to be concentrated on function over form.

I use Bisq daily and feel at home with it's functionality and design. 

The above being said I do think there are a number of improvements that can be made. Initially to [onboard new users easier](https://xd.adobe.com/view/a83c2327-4730-4ec2-8938-e318b2749588-fd6f/). 

Also I think there are very good points discussed on [Improve BSQ account creation & purchase process](https://github.com/bisq-network/bisq/issues/5046). A UI design could help this.

With regards goals here are the ways I see the UI achieving them:

>Enable Bisq to capitalize on it's clear competitive advantages

Include Bisq's branding and key messages within the app. Show users from day one how Bisq is different. I think the website does a good job of this, this could be carried over into the app.

>Improve user conversion rates

There has been a number of discussions in Keybase and Reddit about Bisq users trying to onboard friends and family to Bisq, but they have struggled to achieve it, in part due to the financial issues you mention, but also due to the UI.

>Increase user loyalty and retention

Showing users simple things like their BSQ amount, BTC saved, BTC balance denominated in their local currency etc would all be positive. Also links to Keybase, Reddit, GitHub, YouTube and other Bisq social channels would bring a sense of community that is lacking from the current app.

>Increase growth in number of users

As above, increase number of new users and retain existing users will grow users.

>Maximize revenue for the DAO

As above, every user that downloads Bisq only to never place a trade is lost opportunity for Bisq. New users seeing a lack of liquidity in their market should be encouraged through the design to make an offer. If a larger number of users made offers this would increase user retention and make the app more sticky. 

>Reduce support issues and associated costs

Implementing things like [keybase](https://github.com/bisq-network/bisq/issues/4904) as a support channel and links to the wiki where appropriate would allow users to find answers quicker and ask them in the relevant sections. 

>Improve the trading experiences for users

I like Bisq, I think the trading experience is smooth. But when I started I was a little lost, took me a while to get used to it. If a redesign can help users feel at home sooner this will improve their experience and increase their confidence when making/taking offers.

>Enable Bisq to be designed in a way that meets a variety of user needs and requirements

This could be as simple as having a responsive layout, or allowing users to increase the font sizes. Or it could be allow it to cater to the needs of a BTC seller, XMR trader, sat stacker etc.

**Comment by m52go** [comment]
@jmacxx you make several good points. 

It seems like the UI update project is well on its way to getting started, but I'll throw this idea out anyway: keep the current UI and add a ~400px panel on the right side of the Bisq window (visible by default on new installs) with pointers on getting started, what to do, where to go, etc. If the UI tools allow it, maybe such a thing could be complemented with a UI tour with tooltip balloons or similar.

**Comment by Conza88** [comment]
> I think it doesn't. I'm not vehemently against the UI being redesigned. I wish those doing it the best of luck, and will help where possible. **But I think it is an unnecessary waste of money**, or money spent in the wrong direction.

I think it is **_definitely_** necessary. However, I put it below the other priorities.[https://github.com/bisq-network/bisq/discussions/5271](https://github.com/bisq-network/bisq/discussions/5271 )

_1. API Release
4. Reduce trade protocol to 1 single transaction
5. Increase liquidity by allowing each offer to have multiple payment methods & currencies associated with it
6. Implement new-user-onboarding and new UI design
8. Improve BSQ buying/selling overall process_

> As a reminder, apparently revamping the UI will (per the project description) :
> 
> * Enable Bisq to capitalize on it's clear competitive advantages
_> * **Improve user conversion rates**_
> * Increase user loyalty and retention
_> * **Increase growth in number of users**_
_> * **Maximize revenue for the DAO**
> * **Reduce support issues and associated costs**_
_> * **Improve the trading experiences for users**_
> * Enable is to be designed in a way that meets a variety of user needs and requirements

I think it'd accomplish those in bold pretty clearly. No-one seriously thinks onboarding anyone new to Bisq as an intro to Bitcoin is a good option. I WANT it to be, but its not. That becoming an option instead of having to try make a non-KYC case to try _OVERCOME_ the obvious barriers regarding ease of use etc. 

Ideally its just USE Bisq, and its as easy an option onboarding as any other exchange. Liquidity etc. and all that will then flow (along with other problems addressed above), but more users who stick around, more folks recommending it be used to NEW people who are wanting to ACQUIRE Bitcoin / those more advanced keen to sell. 

A new BSQ buy process is vital for the lifeblood of DAO, and more users, more trades ideally, more BSQ fee's bought/burned. Easier, simpler experience. 

> And finally I think the current UI is really very good.

I have also gotten used to it. But it needs to be more noob, new user friendly. The 'basic' walk-through I suspect might get 'old' for power users/advanced, which if possible I think there should be the option to maintain/revert to old interface (not sure of 'costs' of that in terms of mem siz etc.)

This gives the choice to users, helps new ones, and for those that don't like 'change'.

**Comment by pazza83** [comment]
I have made the proposal comment for voting for the new UI to get started here: https://github.com/bisq-network/projects/issues/49#issuecomment-812277419  Please give a quick thumbs up or thumbs down. On the Github comment. Thanks
